src/lib/ffi/ffi_zfec.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+* (C) 2023 Least Authority TFA GmbH
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/ffi.h>
+#include <botan/zfec.h>
```

Comment:
The include and the functions themselves should be guarded by `BOTAN_HAS_ZFEC`, so that it is possible to compile the C layer even if ZFEC is disabled from the build.

The functions themselves should still be defined mind you! And just return `BOTAN_FFI_ERROR_NOT_IMPLEMENTED` in that case.